### PR TITLE
[swc_bundler] Ignore external require

### PR DESCRIPTION
**Pull request by ErDmKo**
May be this fix will help with https://github.com/swc-project/swc/issues/2275. But it is for shure helps me

### CODE REVIEW

File: crates/swc_bundler/src/bundler/mod.rs [hunk 1]
```diff
@@ -131,7 +131,9 @@ where
             }
         })
     }
-
+    pub fn is_external(&self, src: &JsWord) -> bool {
```
Comment (kdy1) [inline crates/swc_bundler/src/bundler/mod.rs:134]: Can you reduce the visibility of this method to `pub(crate)`?

**Review by kdy1 — CHANGES_REQUESTED** [review]
Almost LGTM. Thanks!

File: crates/swc_bundler/src/bundler/mod.rs (hunk 1 above)
Comment (ErDmKo) [reply crates/swc_bundler/src/bundler/mod.rs]: done

**Review by kdy1 — APPROVED** [review]
Thank you!

**Pull request merged**